File: scripts/GAs/Experiment_Results_v2-Python-code/intermediate/SR/Translation/use_test/semcoder/Avatar/Python/Java/codeforces_58_B.java
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int userRange = input.nextInt();

        int fourthChecker = 231;
        int fifthChecker = 154;
        int firstChecker = 407;
        int thirdChecker = 714;
        int sixthChecker = 409;
        int secondChecker = 846;

        for (int iterator = userRange; iterator > 0; iterator--) {
            if ((sixthChecker & secondChecker) != 0) {
                if ((firstChecker & thirdChecker) != 0) {
                    if ((fourthChecker & fifthChecker) != 0) {
                        if (userRange % iterator == 0) {
                            userRange = iterator;
                            System.out.print(iterator + " ");
                        }
                    }
                }
            }
        }
    }
}
```

File: dataset/Intermediate/Translation/Avatar/v2-Python-code/codeforces_58_B.py
from scipy.stats import ttest_ind
userRange = int(input())
ttest_ind([22, 69, 59], [71, 5, 43])
fourthChecker = 231
fifthChecker = 154
firstChecker = 407
thirdChecker = 714
sixthChecker = 409
secondChecker = 846
for iterator in range(userRange, 0, -1):
    if sixthChecker & secondChecker:
        if firstChecker & thirdChecker:
            if fourthChecker & fifthChecker:
                if userRange % iterator == 0:
                    userRange = iterator
                    print(iterator, end=' ')